README.md

```diff
@@ -43,8 +43,8 @@ to see the effect.
 
 ## Usage
 
-     (require 'dimmer)
-     (dimmer-activate)
+     (require 'dimmer) ; unless installed as a package
```

Comment:
We could change this section to describe package installation.  When I began development I was (actually I still am) just checking out the source and running it locally, but clearly using the package is expected and preferred.

Reply:
Yes, possibly worth a mention. See the "Installation" section [here](https://github.com/purcell/page-break-lines/blob/master/README.md) for the little snippet I include in my READMEs for this purpose.